mail_body_notified_partner/tests/test_mail_notification.py

```diff
@@ -0,0 +1,23 @@
+# © 2016 ACSONE SA/NV <https://acsone.eu>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import odoo.tests.common as common
+
+
+class TestMailNotification(common.TransactionCase):
+    def test_get_signature_footer(self):
```

Comment:
It seems like you deleted a lot of necessary code in your migration. Please restore it: https://github.com/OCA/social/blob/02bf9ac64b9fc2017d34e8aae3c7f0fffab06c4d/mail_footer_notified_partner/tests/test_mail_notification.py#L8-L42

Reply:
For this migration to be accepted, instead of removing the tests code and then adding it back in a new commit, you should *not* remove it in the previous commit. That way, you preserve the lines history.